frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx

```diff
@@ -147,7 +176,7 @@ export const VMDisks: React.FC<VMDisksProps> = ({ vmLikeEntity, pvcs, datavolume
       )}
       <div className="co-m-pane__body">
         <VMDisksTable
-          data={getStoragesData({ vmLikeEntity, pvcs, datavolumes })}
+          data={getStoragesData({ vmLikeEntity, pvcs, datavolumes, templateValidations })}
           customData={{
```

Comment:
lets pass the templateValidations to customData and not bundle it with each disk as it is the same entity